news/_posts/2020-05-11-cms-security-summit.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title:  "CMS Security Summit with Google"
+subtitle: "Security is very important for PrestaShop"
+date:   2020-05-14 17:00:00
+authors: [ PierreRambaud ]
+icon: icon-padlock
+tags: [security]
+---
+
+[Once again](https://build.prestashop.com/news/we-were-at-the-cms-security-summit-with-google/), open source CMSs Security leads shared knowledge together to an event organized by Google in Munich.
+
+# Lightingtalks & Security technologies
+
+After spending the first evening with stakeholders and meeting the newcomers over a few beers, we spent two days at Google Munich office and [Information Security Hub (ISH)](https://www.ish-muc.com/) to speak about what we did since the last years and talk about what's next.
```

Comment:
🍻 

I dont know if mentioning we drink beer is the most important information to put in this sentence which is the 1st one of the article 😅?

Reply:
it's a get-together celebration :sweat_smile:

Reply:
Then OK 👍